src/content/docs/getting-started/install.md

```diff
@@ -2,34 +2,31 @@
 title: Installation
 ---
 
-## Prerequisites
+## Membrane Web IDE
 
-- Git -- [How to install Git](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git)
-- Vscode -- [Download and install VSCode](https://code.visualstudio.com/download)
+Membrane's primary interface is the Membrane Web IDE (fork of VS Code). The IDE is where you deploy Membrane programs, set up cron jobs, manually run programs, inspect logs, and more.
 
-## VSCode Extension
+Visit [membrane.io/ide](https://www.membrane.io/ide) to sign up and open your Membrane workspace.
 
-Membrane's primary interface is its VSCode extension. It handles deploying programs, listing which programs are running, displaying logs, and more.
+<!-- TODO: add gif of web IDE startup -->
```

Comment:
What blank account can I use to screen record? Mine has a bunch of programs that could be a distraction

Reply:
I would just create an empty one. I think you can do that by using `+`. So `pete+demo@membrane.io`.